src/NServiceBus.Core/Persistence/PersistenceDefinition.cs

```diff
@@ -68,6 +68,14 @@ public bool HasSupportFor(Storage storage)
             return storageToActionMap.ContainsKey(typeof(T));
         }
 
+        /// <summary>
+        /// True if supplied storage is supported
+        /// </summary>
+        public bool HasSupportFor(Type storageType) 
```

Comment:
@Particular/nservicebus 
Should this be public?
I can't come up with a reason to expose this to the end users, anyone else?